docs/client/kerberized_kyuubi.md

```diff
@@ -0,0 +1,120 @@
+<!--
+ - Licensed to the Apache Software Foundation (ASF) under one or more
+ - contributor license agreements.  See the NOTICE file distributed with
+ - this work for additional information regarding copyright ownership.
+ - The ASF licenses this file to You under the Apache License, Version 2.0
+ - (the "License"); you may not use this file except in compliance with
+ - the License.  You may obtain a copy of the License at
+ -
+ -   http://www.apache.org/licenses/LICENSE-2.0
+ -
+ - Unless required by applicable law or agreed to in writing, software
+ - distributed under the License is distributed on an "AS IS" BASIS,
+ - WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ - See the License for the specific language governing permissions and
+ - limitations under the License.
+ -->
+
+<div align=center>
+
+![](../imgs/kyuubi_logo.png)
+
+</div>
+
+# Access Kerberized Kyuubi with Beeline & BI Tools
+
+## Instructions
+When Kyuubi is secured by Kerberos, we can not connect to Kyuubi simply by providing a JDBC url 
+with username and password.
+Instead, following steps should be taken.
+
+## Installing and Configuring the Kerberos Clients
+Usually, Kerberos client is installed as default. You can validate it using klist tool.
+
+```bash
+$ klist -V
```

Comment:
It does not work on macOS, `klist -v` works

Reply:
Consider lots of users use BI tools on Windows, should we also give them some instructions?

Reply:
> Consider lots of users use BI tools on Windows, should we also give them some instructions?

Added configuration details for different OS platforms.